loopchain/configure_default.py

```diff
@@ -233,7 +233,9 @@ class ApiVersion(IntEnum):
 BLOCK_SYNC_RETRY_NUMBER = 5
 TIMEOUT_FOR_LEADER_COMPLAIN = 60
 MAX_TIMEOUT_FOR_LEADER_COMPLAIN = 300
-
+CITIZEN_ASYNC_RESULT_MAX_SIZE = 10_000_000 # dictionary Length
+CITIZEN_ASYNC_REQUEST_RETRY_TIMES = 20 # times
```

Comment:
I think anyone can assume this configuration that means `times` by name without the comment.